civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -6312,11 +6312,18 @@ function civihr_employee_portal_hrreport_get_exposed_filter_default_values($view
   if (!empty($dateFilter['expose']['identifier'])) {
     $identifier = $dateFilter['expose']['identifier'];
 
-    $defaultFromDate = !empty($dateFilter['default_date']) ? $dateFilter['default_date'] : '';
-    $defaultToDate = !empty($dateFilter['default_to_date']) ? $dateFilter['default_to_date'] : '';
+    $defaultFromDate = !empty($dateFilter['default_date']) ? $dateFilter['default_date'] : null;
```

Comment:
and to avoid duplicated you could separate this into a new function and call it twice for min and max values